app/models/topical_event.rb

```diff
@@ -63,4 +71,19 @@ def ordered_featured_documents
       }
     end
   end
+
+private
+
+  def fetch_related_publications
+    @documents_service.fetch_related_documents_with_format({ filter_format: "publication" })
+  end
+
+  def fetch_related_consultations
+    @documents_service.fetch_related_documents_with_format({ filter_format: "consultation" })
+  end
+
+  def fetch_related_announcements
+    announcement_document_types = %w[press_release news_article news_story fatality_notice speech written_statement oral_statement authored_article government_response]
```

Comment:
A really small nitpick, but I think these would be more aesthetically pleasing if each item were on a separate line (and it'd make the Git diff clearer if we update this list in the future).